### PR TITLE
TCDTF Poland

### PR DESCRIPTION
**Pull request by Fullchord**
-rebalanced ai, and player Poland starts
-made game rule no longer require a player
-changed a few Poland focuses to better templates

### CODE REVIEW

**Review by Juddur — APPROVED** [review]
Just redoes what poland already does

**Review by xfrenzzyy — COMMENTED** [review]
Seems like another Poland AI buff unless I'm reading these changes wrong.

File: common/on_actions/pol_on_actions.txt [hunk 1]
```diff
@@ -5,17 +5,13 @@ on_actions = {
...
-						
-						POL = { is_ai = no }
+					has_game_rule = {
+						rule = POL_balance_selection
+						option = PLAYER_POLAND
 					}
 				}
 				POL = {
-					load_oob = "POL_1936_player"
+					load_oob = "POL_1936_ai"
```
Comment (atomicplan666) [inline common/on_actions/pol_on_actions.txt:14]: If has game rule player poland then load ai oob? doesn't make sense to me

File: common/on_actions/pol_on_actions.txt [hunk 2]
```diff
@@ -43,7 +39,7 @@ on_actions = {
 			}
 			else = {
 				POL = {
-					load_oob = "POL_1936_ai"
+					load_oob = "POL_1936_player"
```
Comment (atomicplan666) [inline common/on_actions/pol_on_actions.txt:42]: backwards?

File: history/units/POL_1936_player.txt [hunk 3]
```diff
@@ -312,6 +316,14 @@ units = {
 		start_experience_factor = 0.3
 		start_equipment_factor = 0.3 #0.6
 
+	}
+	division= {
```
Comment (atomicplan666) [inline history/units/POL_1936_player.txt:320]: Why are there brackets added and deleted in this way?

**Review by atomicplan666 — CHANGES_REQUESTED** [review]

File: common/on_actions/pol_on_actions.txt (hunk 1 above)
Comment (Endsiegg) [reply common/on_actions/pol_on_actions.txt:14]: Seems like it backwards

File: common/national_focus/poland.txt [hunk 4]
```diff
@@ -1938,7 +1938,7 @@ focus_tree = {
 		completion_reward = {
 			add_equipment_to_stockpile = {
 				type = infantry_equipment
-				amount = 2000
+				amount = 3000
```
Comment (Endsiegg) [inline common/national_focus/poland.txt:1941]: AI does not need more equipment.

File: common/national_focus/poland.txt [hunk 5]
```diff
@@ -5809,6 +5810,7 @@ focus_tree = {
 					mountaineers = { x = 2 y = 2 }
 				}
 				support = {
+					artillery = { x = 0 y = 1 }
```
Comment (Endsiegg) [inline common/national_focus/poland.txt:5813]: More Arty?

File: common/on_actions/pol_on_actions.txt [hunk 6]
```diff
@@ -30,7 +26,7 @@ on_actions = {
 					
 					add_equipment_to_stockpile = {
 						type = infantry_equipment_0
-						amount = 11000
+						amount = 13000
```
Comment (Endsiegg) [inline common/on_actions/pol_on_actions.txt:29]: so now 4k more equipment?

File: history/units/POL_1936_player.txt [hunk 7]
```diff
@@ -171,21 +167,21 @@ units = {
 		location = 3320
 		division_template = "Korpus Ochrony Pogranicza"
 		start_experience_factor = 0.81
-		start_equipment_factor = 0.4 #0.8
+		start_equipment_factor = 0.8
```
Comment (Endsiegg) [inline history/units/POL_1936_player.txt:170]: Does not need double starting, we talked about this being an issue.

File: history/units/POL_1936_player.txt [hunk 8]
```diff
@@ -171,21 +167,21 @@ units = {
...
 	}
 	division= {
 		name = "KOP Zdołbunów"
 		location = 572
 		division_template = "Korpus Ochrony Pogranicza"
 		start_experience_factor = 0.81
-		start_equipment_factor = 0.4 #0.8
+		start_equipment_factor = 0.8
```
Comment (Endsiegg) [inline history/units/POL_1936_player.txt:184]: Same as above

File: history/units/POL_1936_player.txt [hunk 9]
```diff
@@ -171,21 +167,21 @@ units = {
...
 	}
 	division= {
 		name = "KOP Snów"
 		location = 3309
 		division_template = "Korpus Ochrony Pogranicza"
 		start_experience_factor = 0.81
-		start_equipment_factor = 0.4 #0.8
+		start_equipment_factor = 0.8
```
Comment (Endsiegg) [inline history/units/POL_1936_player.txt:177]: Same as above

File: history/units/POL_1936_player.txt [hunk 10]
```diff
@@ -205,86 +201,78 @@ units = {
...
+		start_equipment_factor = 0.8 #0.3
+	}
+	division= {	#Garrisoned at Przemysl
+		name = "10 Grupa Artylerii"
+		location = 9494
+		division_template = "Grupa Artylerii"
+		force_equipment_variants = { infantry_equipment_0 = { owner = "POL" } }
+		start_experience_factor = 0.45
+		start_equipment_factor = 0.8 #0.3
+	}
```
Comment (Endsiegg) [inline history/units/POL_1936_player.txt:275]: Why the hell are these doubled???

File: history/units/POL_1936_player.txt [hunk 11]
```diff
@@ -294,15 +282,31 @@ units = {
...
-		start_equipment_factor = 0.3 #0.6
+		start_equipment_factor = 0.8 #0.6
+
+	}
+	division= {	
+		name = "98 Dywizja Piechoty"
+		location = 362
+		division_template = "Dywizja Piechoty"
+		start_experience_factor = 0.3
+		start_equipment_factor = 0.8 #0.6
```
Comment (Endsiegg) [inline history/units/POL_1936_player.txt:293]: Again? wtf is your obsession with buffing poland

File: history/units/POL_1936_player.txt [hunk 12]
```diff
@@ -294,15 +282,31 @@ units = {
...
 
 	}
 	division= {	
 		name = "15 Dywizja Piechoty"
 		location = 389
 		division_template = "Dywizja Piechoty"
 		start_experience_factor = 0.3
-		start_equipment_factor = 0.3 #0.6
+		start_equipment_factor = 0.8 #0.6
```
Comment (Endsiegg) [inline history/units/POL_1936_player.txt:301]: 0.3 to 0.8 What in the world

File: history/units/POL_1936_player.txt [hunk 13]
```diff
@@ -320,23 +332,31 @@ units = {
 		location = 6558
 		division_template = "Dywizja Piechoty"
 		start_experience_factor = 0.76
-		start_equipment_factor = 0.3 #0.6
+		start_equipment_factor = 0.6
```
Comment (Endsiegg) [inline history/units/POL_1936_player.txt:335]: I see doubles everywhere, stop secretly buffing shit

**Review by Endsiegg — CHANGES_REQUESTED** [review]
UNDO ALL SUPER BUFFS TO POLAND AGAIN

**Comment by atomicplan666** [comment]
This update essentially removes the option for hosts to debuff Poland. Why would a change like this be needed?

**Comment by atomicplan666** [comment]
> This update essentially removes the option for hosts to debuff Poland. Why would a change like this be needed?

Not like hosts could even control Poland to begin with. Which hosts should be allowed to do.

**Comment by Fullchord** [comment]
People did not like the chnages

**Pull request closed without merging**